src/sentry/workflow_engine/models/data_condition.py

```diff
@@ -19,6 +19,10 @@
 logger = logging.getLogger(__name__)
 
 
+class ConditionType(StrEnum):
+    METRIC_CONDITION = "MetricCondition"
```

Comment:
This can be added to the `Condition` str enum below, like the `GROUP_EVENT_ATTR_COMPARISON`.

FYI -- I'm going to put together a PR for the data_conditions in general to update some of the column names an improve all these hooks since it's feeling a bit all over right now.

Reply:
I'm confused what the difference between a `DataCondition` `condition` and `type` is. `condition` has a comment saying it's gt, lt, eq, etc. which corresponds to the `Condition` enum but I created `ConditionType` for the `type`, and it sounds like you're saying they take the same value, is that right?

Reply:
They're _close_ to being the same, in a highly confusing way. That's why I'd like to do a quick update to the model to rename a few fields and add the typing separately.